publishers:
  asr:
    topic: /xf_ros/asr_msg
    queue_size: 3
    latch: false
